
bool enable_taxi
float32 des_x
float32 des_y
bool enable_userCtrl
bool enable_helper

